package.json

```diff
@@ -25,8 +25,9 @@
     "test": "npm run check && vitest run",
     "e2e": "cypress open --e2e",
     "lint": "eslint . --ext .js,.ts,.svelte --ignore-path .gitignore",
-    "prepare": "is-ci || husky install",
-    "prepublishOnly": "is-ci || npm run build"
+    "prepare": "ts-patch install -s && is-ci || husky install",
+    "prepublishOnly": "is-ci || npm run build",
+    "publish:npm": "npm publish --access public"
```

Comment:
is this needed? the default access is public. isn't it?

Reply:
Yes I suppose it defaults to public, I put it in there to be explicit, happy to take the access part out